packages/shopify-checkout/src/utils/buildCheckout.ts

```diff
@@ -2,11 +2,16 @@ import { BuildCheckoutParams, ShopifyCheckout } from '../checkout-client.types';
 
 export default function buildCheckout({
   id,
-  webUrl
+  webUrl,
+  completedAt,
+  lineItems,
+  discountApplications
 }: BuildCheckoutParams): ShopifyCheckout {
   return {
     id,
     url: webUrl,
-    completed: false
+    completed: Boolean(completedAt),
+    lines: lineItems.edges.map((edge) => edge.node),
+    discounts: discountApplications.edges.map((edge) => edge.node)
```

Comment:
Could we write tests for these lines? This file has pretty low coverage (probably cause it's so small) and running the test points to these two lines.